library/src/app/components/transfer/transfer.component.html

```diff
@@ -0,0 +1,141 @@
+<ng-container *ngIf="(isLoading$ | async) !== true; else loading">
+  <ng-container
+    *ngIf="{
+      fiatAccount: fiatAccount$ | async,
+      bankAccounts: bankAccounts$ | async
+    } as accounts"
+  >
+    <ng-container *ngIf="configService.getConfig$() | async as config">
+      <app-navigation [routingData]="routingData"></app-navigation>
+      <ng-container *ngIf="accounts.bankAccounts!.length > 0; else noAccount">
+        <div class="cybrid-balance">
+          <span class="mat-hint">{{
+            'accountList.available' | translate
+          }}</span>
+          <div>
+            <h1>
+              {{
+                accounts.fiatAccount?.platform_available!
+                  | asset : fiatAsset : 'formatted'
+              }}
+              <span class="mat-hint">{{
+                accounts.fiatAccount?.asset | uppercase
+              }}</span>
+            </h1>
+          </div>
+        </div>
+        <mat-tab-group
+          animationDuration="0ms"
+          (selectedTabChange)="onSwitchSide($event.tab.origin)"
+        >
+          <mat-tab
+            label="{{ 'transfer.deposit' | translate | uppercase }}"
+          ></mat-tab>
+          <mat-tab
+            label="{{ 'transfer.withdraw' | translate | uppercase }}"
+          ></mat-tab>
+        </mat-tab-group>
+        <ng-container [formGroup]="transferGroup">
+          <div class="cybrid-tabs">
+            <div class="cybrid-input-wrapper">
+              <div class="cybrid-input-flex">
+                <label class="mat-hint cybrid-h5">{{
+                  'transfer.bankAccount' | translate
+                }}</label>
+                <mat-form-field appearance="outline">
+                  <mat-select #account formControlName="account">
+                    <mat-select-trigger>
+                      <div class="cybrid-option">
+                        <mat-icon>savings</mat-icon>
+                        <span class="cybrid-trigger-name">
+                          <strong>{{
+                            account.value.plaid_institution_id + ' - '
```

Comment:
For now this is going to come back as an identifier. I'll need to add code to resolve the ID to a pretty printable name.